examples/java/src/main/java/glide/examples/StandaloneExample.java

```
@@ -0,0 +1,128 @@
+     *
+     * @param client An instance of <code>GlideClient</code>.
+     */
+    public static void appLogic(GlideClient client) {
+
+        // Send SET and GET
+        try {
+            CompletableFuture<String> setResponse = client.set("foo", "bar");
+            Logger.log(Logger.Level.INFO, "app", "Set response is " + setResponse.get());
+        } catch (ExecutionException | InterruptedException e) {
+            Logger.log(Logger.Level.ERROR, "glide", "Error during SET: " + e.getMessage());
+        }
+
+        try {
+            CompletableFuture<String> getResponse = client.get("foo");
+            Logger.log(Logger.Level.INFO, "app", "Get response is " + getResponse.get());
+        } catch (ExecutionException | InterruptedException e) {
+            Logger.log(Logger.Level.ERROR, "glide", "Error during GET: " + e.getMessage());
+        }
+
+        // Send PING to the primary node
+        try {
+            CompletableFuture<String> pong = client.ping();
+            Logger.log(Logger.Level.INFO, "app", "Ping response is " + pong.get());
+        } catch (ExecutionException | InterruptedException e) {
+            Logger.log(Logger.Level.ERROR, "glide", "Error during PING: " + e.getMessage());
+        }
+    }
+
+    /** Executes the application logic with exception handling. */
+    private static void execAppLogic() {
+
+        while (true) {
+            GlideClient client = null;
+            try {
+                client = createClient();
+
+                assert client != null;
+                appLogic(client);
+                return;
+
+            } catch (ClosingException e) {
+                // If the error message contains "NOAUTH", raise the exception
+                // because it indicates a critical authentication issue.
+                if (e.getMessage().contains("NOAUTH")) {
+                    Logger.log(
+                            Logger.Level.ERROR, "glide", "Authentication error encountered: " + e.getMessage());
```

The comment mentions we should raise the exception but we don't do that here, should we add a throw here?
```suggestion
                            Logger.Level.ERROR, "glide", "Authentication error encountered: " + e.getMessage());
                            throw e;
```

---

Looks like we added and then removed the `throw e`. Shouldn't we throw here like the comment indicates?